templates/app/views/autocomplete_results/_autocomplete_results.html.erb

```diff
@@ -0,0 +1,36 @@
+<ul class="autocomplete-results__list">
+  <li class="autocomplete-results__title">
+    <%= t('spree.products') %>
+  </li>
+
+  <% if results[:products].any? %>
+    <% results[:products].each do |product| %>
+      <li class="autocomplete-results__item"
+          data-search-target="result">
+        <%= link_to product.name, product_path(product) %>
+      </li>
+    <% end %>
+
+    <li class="autocomplete-results__item"
+        data-search-target="result">
+      <%= link_to "See all results", products_path(keywords: params[:keywords]) %>
+    </li>
+  <% else %>
+    <li class="autocomplete-results__item autocomplete-results__item--empty">No results</li>
+  <% end %>
```

Comment:
Looks like the only difference between this block and the following one is the translation key and the results url/name. Thoughts on refactoring that to a loop or partial?

Reply:
One difference is that the products list has the `See all results` item while the taxons list doesn't (taxons don't have an index page, filterable by a search term). 

That was enough for me to determine that an abstraction here wasn't the right thing, but let me know if you think we should move something in a partial anyway.